src/midi/midi.cpp

```diff
@@ -272,7 +277,11 @@ class MIDI : public Module_base {
 			if (handler->Open(conf)) {
 				midi.available=true;
 				midi.handler=handler;
-				LOG_MSG("MIDI: Opened device: %s", name.c_str());
+				if (name == "none")
+					LOG_MSG("MIDI: No device selected - output disabled");
```

Comment:
(same here).

Reply:
The function is quite bloated though and I know the plan is to refactor this. 
(I've refactored it in my open-async PR, but that's just one attempt). So we could also do more output cleanup in that future PR too.

Reply:
(I dropped the change for now - to be addressed in another PR)